hazelcast/src/test/java/com/hazelcast/genericrecord/GenericRecordBuilderTest.java

```diff
@@ -101,4 +101,20 @@ public void run() {
             }
         });
     }
+
+    @Test
+    public void testUnwrittenFieldsThrowException() {
+        ClassDefinition classDefinition =
+                new ClassDefinitionBuilder(TestSerializationConstants.PORTABLE_FACTORY_ID, TestSerializationConstants.NAMED_PORTABLE)
+                        .addUTFField("name").addIntField("myint").build();
+
+        GenericRecord.Builder builder = GenericRecord.Builder.portable(classDefinition);
+        builder.writeInt("myint", 1);
+        assertThrows(HazelcastSerializationException.class, new Runnable() {
```

Comment:
nit this assertion (and other similar ones in this test) can be simplified to `assertThrows(HazelcastSerializationException.class, builder::build);`